tests/test_s3_to_json.py

```diff
@@ -105,6 +105,7 @@ def test_write_healthkitv2samples_file_to_json_dataset(self, s3_obj, namespace,
                 dataset_identifier="HealthKitV2Samples",
                 metadata=sample_metadata["Metadata"],
                 workflow_run_properties=workflow_run_properties,
+                delete_upon_successful_upload=False,
```

Comment:
Do you think we need a test that tests that `os.remove` does get called as the default for `write_file_to_json_dataset` just to make sure that's always true?

Reply:
Yeah, I'll add it.